### PR TITLE
Remove command lock without initialization

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**

Removed the block on using commands when not adding a commons repository.

Now only the `rit create formula` command will remain blocked, other core commands or commands added via the repository will function normally.

>aa

**- How to verify it**

> Clone this pr with the command and build Ritchie
```bash
git fetch upstream pull/496/head:improvement/change-command-locking-without-rit-init
git checkout improvement/change-command-locking-without-rit-init
make build && sudo cp dist/linux/rit /usr/local/bin/
```
> And run rit add repo
```bash
rit list repo # It is not blocked
rit create formula # Is blocked
```

**- Description for the changelog**

Remove blocking of most commands when no commons repository is added

### CODE REVIEW

File: pkg/cmd/root.go [hunk 1]
```diff
@@ -62,6 +62,10 @@ var (
 	upgradeList = []string{
 		cmdUse,
 	}
+
+	listCommandsBlockedByCommons = []string{
```
Comment (kaduartur) [inline pkg/cmd/root.go:66]: Could you rename the variable to `blockedCmdsByCommons`?
What do you think about it?

**Review by kaduartur — CHANGES_REQUESTED** [review]
LGTM! 🚀

File: pkg/cmd/root.go (hunk 1 above)
Comment (brunasilvazup) [reply pkg/cmd/root.go:66]: I think it's a good idea.

File: Makefile [hunk 2]
```diff
@@ -118,7 +118,10 @@ unit-test:
 
 functional-test:
 	mkdir -p $(BIN)
-	$(GO_TEST) -v -count=1 -p 1 `go list ./functional/... | grep -v vendor/`
+	$(GO_TEST) -v -count=1 -p 1 `go list ./functional/init/... | grep -v vendor/`
```
[the remark this answers is not in the document]
Comment (brunasilvazup) [reply Makefile:121]: I just don't know if it would be clear the need for the core tests to be last, but I think it is a valid solution today.

**Review by kaduartur — APPROVED** [review]

**Review by victor-schumacher — APPROVED** [review]

**Review by lucasdittrichzup — APPROVED** [review]

**Pull request merged**